### DISCUSSION TITLE
Bug introduced in FastAPI 0.111.1

### DISCUSSION THREAD

**Discussion by s-rigaud** [fastapi/fastapi#11845] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
@router.get("/")
def filter_models(
    bizdev_ids: list[int] | None = Query(
        title="BizDev user ids", example="42", default=None
    ),
    pm_ids: list[int] | None = Query(title="PM user ids", example="36", default=None),
    designer_ids: list[int] | None = Query(
        title="Designer user ids", example="666", default=None
    ),
) :
    """Endpoint o.1"""
    pass
```

### Description

@tiangolo (sorry for the ping but it seems like a bug directly introduced during your last release)

Due to Swagger version restriction removed, the new version is used. For the example the UI moved from :

![image](https://github.com/user-attachments/assets/efe7774a-b1b8-4011-8a0d-8855206c6b24)

to 

![image](https://github.com/user-attachments/assets/0cfbeb4c-d08c-49cf-810d-dd54766f8fe7)

but those fields have actually some hidden values as you can see in the request while nothing seems to be shown in the UI.

![image](https://github.com/user-attachments/assets/4388b86d-a4f2-44b2-aee2-175b89ffcf42)
 

### Operating System

Linux

### FastAPI Version

0.111.1

### Pydantic Version

2.8.2

### Python Version

3.11

**Comment by YuriiMotov** [comment, marked as answer]
This is not a bug. In the new version you can see correct types of Query parameters and everything seem to work as it should.

The only problem is that it doesn't show your examples, isn't it?
To specify examples you should use `openapi_examples`:
```
def filter_models(
    bizdev_ids: list[int] | None = Query(
        title="BizDev user ids", openapi_examples={"example 1": {"value": [42]}}, default=None
    ),
    pm_ids: list[int] | None = Query(title="PM user ids", openapi_examples={"Example 1": {"value": [36, 47]}}, default=None),
    designer_ids: list[int] | None = Query(
        title="Designer user ids", openapi_examples={"example 1": {"value": [666, 777]}}, default=None
    ),
) :
    """Endpoint o.1"""
    pass
```

**Reply by notypecheck** [reply]
I think examples work as expected but they were trying to specify `int` example for a `list[int]` type
